### PR TITLE
[Patterns]: Update Query Loop patterns with less posts per page

### PR DESCRIPTION
**Pull request by ntsekouras**
Trac ticket: https://core.trac.wordpress.org/ticket/54639

I think we need to 'lighten' the Query Loop patterns regarding the number of posts it previews. The point of a pattern is to showcase some design and with some pre-selected blocks(the inner Post blocks, etc..). It's already opinionated about the number of posts, so we could just show the minimum number required to make the pattern show its point.

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by SergeyBiryukov — APPROVED** [review]

**Comment by ntsekouras** [comment]
Thanks for reviewing @SergeyBiryukov , but let's wait just a bit before merging as there were some concerns raised about this by @mtias here: https://github.com/WordPress/gutenberg/pull/37475#issuecomment-996534835.

There is the possibility of reverting the GB PR and that would make this PR obsolete.

**Review by mcsf — CHANGES_REQUESTED** [review]
Flagging this just to make sure we don't prematurely merge it, per the following:

> Thanks for reviewing […] but let's wait just a bit before merging as there were some concerns raised about this […] here: WordPress/gutenberg#37475 (comment).
> 
> There is the possibility of reverting the GB PR and that would make this PR obsolete.

**Comment by mcsf** [comment]
^ I was hoping that my review at https://github.com/WordPress/wordpress-develop/pull/2063#pullrequestreview-835446418 would add a red "changes requested" mention by the merge button. It apparently didn't, so merger beware. :)

**Comment by ntsekouras** [comment]
I'm closing this as we reverted the GB PR eventually.

**Pull request closed without merging**
